code-goodbuy/goodbuy-nextjs
# Refresh token & Refactoring

### What's new

- Added api routes for login, logout, and token refresh
- API tests

### What changed

- Login tests
- Auth logic has been moved
- Refactored forms
- Refactored navbar
- Refactored API
- Refactored contexts
- Fixed bugs in the landing page

### How to test

> I know that there is a lot of new code by running the tests and running this locally should make you confident that this works how it should. Much of the "new" code is old code refactored.

- Download/pull the code
- Run ```git checkout refresh_token```
- Run ```npm i```
- Run ```npm run test```
- [Optional] run ```npm run dev``` if you want to run this locally
